_includes/metadata.html

```diff
@@ -0,0 +1,142 @@
+<!-- Standard metadata -->
+{%- if site.enable_google_verification -%}
+<meta name="google-site-verification" content="{{ site.google_site_verification }}" />
+{%- endif -%}
+{%- if site.enable_bing_verification -%}
+<meta name="msvalidate.01" content="{{ site.bing_site_verification }}" />
+{%- endif -%}
```

Comment:
question: wouldn't adding `-` to both sides of `{%-` and `-%}` result in having both `<meta />` tags on the same line? (this is a cool pattern btw, didn't know about it)

Reply:
I have had to nest conditionals to keep the two labels on the same line, also when only one of them is activated.

```liquid
{%- if site.enable_google_verification or site.enable_bing_verification %}
    <!-- Website verification -->
    {% if site.enable_google_verification -%}
    <meta name="google-site-verification" content="{{ site.google_site_verification }}" />
    {%- endif -%}
    {% if site.enable_bing_verification -%}
    <meta name="msvalidate.01" content="{{ site.bing_site_verification }}" />
    {%- endif -%}
{%- endif %}
```

I have also left the HTML comments inside the conditionals, so that they are only printed if the site verification, opengraph, etc. is generated.

PR: https://github.com/alshedivat/al-folio/pull/481/commits/5dfa4447ef746e3dd0418e989c32f52bb20813bb

The final result with data is:
<img width="1475" alt="Screenshot of Google Chrome (22-12-21, 10-47-55)" src="https://user-images.githubusercontent.com/28966312/147072905-be3794e4-9f63-44b9-b97b-d7c74da4cb4e.png">